cogs/FunAndGames.py

```diff
@@ -67,5 +67,41 @@ def __init__(self, bot):
             else:
                 returnChar = returnChar + character.lower()
         await ctx.send(returnChar)
-
+        
     
+    @commands.command(name='uwu')
+    async def uwu(self, ctx, arg=None):
+        if(arg == None):
+            history = await ctx.history(limit=2).flatten()
+            lastMsg = history[1].content
+            uwuMsg = makeUwUText(self, lastMsg)
+        else:
+            uwuMsg = makeUwUText(self, arg)
+        await ctx.send(uwuMsg)
+            
+    @uwu.error
+    async def uwu_error(self, ctx, error):
+        await ctx.send(f"OwO Oh Noes, Wooks wike an ewwow occuwed. Pwease dwon't hwate mwe .·´¯`(>▂<)´¯`·.\n f{error}")
+
+def makeUwUText(self, lastMsg):
```

Comment:
you can still make this part of the cog without using the @command rapper, just indent it and change it to "async def"

Reply:
highlighted extra lines by mistake

Reply:
Hmmm, it doesn't seem to like it when I change it like that